frontend/src/components/regions/UsersAndGroups/UsersAndGroups.test.js

```diff
@@ -72,3 +73,57 @@ test("cleanAndFilterTeams can deal with multiple words in any order", () => {
   const names = ret.map((group) => group.name);
   expect(names).toEqual(["demo group 1 AAA", "demo group 2 aaa"]);
 });
+
+test("getPrStatus function returns error string when the pr age is 2 days or older", () => {
+  const date = new Date();
+  const card = {
+    oldestOpenPrUpdatedTime: date.setDate(date.getDate() - 3),
```

Comment:
Please dopnt hard code numbers.

If I decided to change the configuration like this then your tests should still pass. 
```
const PR_WARNING_AGE_THRESHOLD = 100
const PR_ERROR_AGE_THRESHOLD = 200
const TILDE_ERROR_AGE_THRESHOLD = 300
const TILDE_WARNING_AGE_THRESHOLD = 200
```

Configuration changes should be allowed. Changing configuration should not break tests.

Please actually change the configuration, make sure your tests pass, then change the configuration values back again.